### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by redhatrises**
- yamllint files and add GH action
- Update main readme and add link to role readme
- Remove unused travis.yml

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -34,7 +35,14 @@ license_file: LICENSE
...
 # requirements as 'namespace' and 'name'
-tags: []
+tags:
+  - crowdstrike
+  - falcon
+  - sensor
+  - linux
+  - windows
+  - macOS
+  - security
```
Comment (shawndwells) [inline galaxy.yml:45]:
```suggestion
  - crowdstrike
  - falcon
  - sensor
  - linux
  - windows
  - macOS
  - security
  - edr
  - xdr
```

Comment (shawndwells) [reply galaxy.yml:45]: Should we add EDR? Yahhh it's a little markety but why not?

File: roles/falcon_installation/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Ansible 2.10 or higher
 Role Variables
 --------------
 
-Currently the following variables are supported:
+Currently, the following variables are supported:
```
Comment (shawndwells) [inline roles/falcon_installation/README.md:14]:
```suggestion
The following variables are currently supported:
```

File: roles/falcon_installation/README.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ This example installs the Falcon Sensor:
 ---
 - hosts: all
   roles:
-  - role: falcon_installation
+  - role: crowdstrike.falcon.falcon_installation
   vars:
     falcon_api_client_id: <Falcon_UI_OAUTH_client_id>
     falcon_api_client_secret: <Falcon_UI_OAUTH_client_secret>
```
Comment (shawndwells) [inline roles/falcon_installation/README.md:58]: Wasn't there some informal naming scheme for these variables?

File: roles/falcon_installation/README.md (hunk 2 above)
Comment (redhatrises) [reply roles/falcon_installation/README.md]: Fixed

File: galaxy.yml (hunk 1 above)
Comment (redhatrises) [reply galaxy.yml:45]: Sure... added

File: roles/falcon_installation/README.md (hunk 3 above)
Comment (redhatrises) [reply roles/falcon_installation/README.md:58]: There is. These were agreed to with @isimluk

**Pull request merged**